### ISSUE TITLE
Migrate Template Hibernate Mapping to DotConnect

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#18730]
- [x] We need to remove the template from hibernate mapping
- [x] We need to use a db transformer to load templates

 We need to test:
* CRUD templates advanced and design
* CRUD pages
* CRUD Containers (focus on containers that are already associated with a template)
* PP templates
* PP pages
* Edit a template (layout)
* Make sure the starter.zip import/export still works (create a backup and use it as starter)

**Comment by erickgonzalez** [comment]
PR: https://github.com/dotCMS/core/pull/19482

**Comment by erickgonzalez** [comment]
We have some issues saving containers:
```
11:23:11.533  WARN  action.EditContainerAction - com.dotmarketing.business.DotStateException: Template mapping was deleted from hibernate files
com.dotmarketing.business.DotStateException: Template mapping was deleted from hibernate files
        at com.dotmarketing.factories.InodeFactory.getChildrenClass(InodeFactory.java:248) ~[dotcms_master_5e3e167.jar:?]
        at com.dotmarketing.factories.InodeFactory.getChildrenClass(InodeFactory.java:234) ~[dotcms_master_5e3e167.jar:?]
        at com.dotmarketing.portlets.containers.action.EditContainerAction._saveWebAsset(EditContainerAction.java:526) ~[dotcms_master_5e3e167.jar:?]
        at com.dotmarketing.portlets.containers.action.EditContainerAction.processAction(EditContainerAction.java:157) ~[dotcms_master_5e3e167.jar:?]
        at com.liferay.portal.struts.PortletRequestProcessor.process(PortletRequestProcessor.java:124) ~[dotcms_master_5e3e167.jar:?]
        at com.liferay.portlet.StrutsPortlet.processAction(StrutsPortlet.java:120) ~[dotcms_master_5e3e167.jar:?]
        at com.liferay.portlet.ConcretePortletWrapper.processAction(ConcretePortletWrapper.java:97) ~[dotcms_master_5e3e167.jar:?]
        at com.liferay.portal.action.LayoutAction._processPortletRequest(LayoutAction.java:194) ~[dotcms_master_5e3e167.jar:?]
        at com.liferay.portal.action.LayoutAction._processActionRequest(LayoutAction.java:206) ~[dotcms_master_5e3e167.jar:?]
        at com.liferay.portal.action.LayoutAction.execute(LayoutAction.java:82) ~[dotcms_master_5e3e167.jar:?]

```

We need to test also CRUD Containers.

**Comment by erickgonzalez** [comment]
PR for new cases found: https://github.com/dotCMS/core/pull/19530

**Comment by erickgonzalez** [comment]
Tested and working as it should. We have a `WARN  db.LocalTransaction - Transaction broken - Connection that started the transaction is not the same as the one who is commiting` that will be taken care when we migrate the template to the new portlet.

**Comment by bryanboza** [comment]
Fixed, tested on release-20.11 // Postgres // FF
